src/server/index.tsx

```diff
@@ -267,3 +301,63 @@ function removeParam(url: string, parameter: string): string {
     .replace(new RegExp("[?&]" + parameter + "=[^&#]*(#.*)?$"), "$1")
     .replace(new RegExp("([?&])" + parameter + "=[^&]*&"), "$1");
 }
+
+const iconSizes = [72, 96, 128, 144, 152, 192, 384, 512];
+const defaultLogoPathDirectory = path.join(
+  process.cwd(),
+  "dist",
+  "assets",
+  "icons"
+);
+
+export async function generateManifestBase64(site: Site) {
```

Comment:
We should probably re-organize some of this into separate files... not necessary for this PR tho.